README.md

```diff
@@ -36,6 +36,22 @@ npm install --global @ui5/linter
 npm install --save-dev @ui5/linter
 ```
 
+## Preparation
```

Comment:
This is not required for end-users. It is only needed for maintainers. Please make this clear and move the section accordingly. Right now I expect many people to get confused by this as it is not relevant for them at all.

Reply:
Thanks!

I hope it's better now and won't confuse the end users of the UI5Linter

@KlattG what do you think? Is it clear enough?

Reply:
+1